.github/workflows/tests.yml

```diff
@@ -36,41 +36,27 @@ jobs:
           python-version: [3.7, 3.8, 3.9]
           requirements-level: [pypi]
           cache-service: [redis]
-          db-service: [postgresql10, postgresql13, mysql5, mysql8]
+          db-service: [postgresql10, postgresql13]
```

Comment:
@zzacharo, please check this change!  The main issue is that mysql is not happy with the table of `vocabularies`.  If I get it right, the trigger for this is that now the module also depends in `invenio-administration`, and that brings its dependencies (`invenio-vocabularies` in particular). Note that  neither`invenio-administration`  nor `invenio-vocabularies` do not check the mysql pipelines. I have the feeling that they did, they would have the same issue.

Reply:
Following @jrcastro2 suggestion, another option would be to put something like https://github.com/inveniosoftware/invenio-vocabularies/pull/261/